models.py

```diff
@@ -249,6 +249,7 @@ def FetchAllEntities(cls):
 class Dayparts(ndb.Model):
   """Examples: Monday AM, or M-W-F 8am-9am"""
   data = ndb.TextProperty()
+  jdata = ndb.JsonProperty()
```

Comment:
I'm surprised this works. So you changed the class by adding a new property. Can you still read the old data that was written before jdata existed? I was imagining you would have to create a whole new class called Dayparts2. Have you tested this?